core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -3235,28 +3096,33 @@ public void activeSyncMetadata(AlluxioURI path, Collection<AlluxioURI> changedFi
     try (RpcContext rpcContext = createRpcContext()) {
       if (changedFiles == null) {
         // full sync
-        LockingScheme lockingScheme = new LockingScheme(path, LockPattern.READ, true);
-        try (LockedInodePath inodePath =
-            mInodeTree.lockInodePath(lockingScheme.getPath(), lockingScheme.getPattern())) {
-          syncMetadataInternal(rpcContext, inodePath, lockingScheme, DescendantType.ALL,
-              populateStatusCache(path, DescendantType.ALL));
+        long start = System.currentTimeMillis();
+
+        // Set sync interval to 0 to force a sync.
+        FileSystemMasterCommonPOptions options =
```

Comment:
Should we make the force sync option a static constant, and reuse it here and below?